lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -740,6 +797,88 @@ ParseResult FIRParser::parseOptionalRUW(RUWAttr &result) {
   return success();
 }
 
+ParseResult FIRParser::importAnnotations(const SMLoc &loc,
```

Comment:
SMLoc is trivial and can be passed by value instead of `const&`.

Reply:
Good point. Fixed in a `!fixup` commit.